packages/turbo/ios/RNVisitableView.swift

```diff
@@ -8,98 +8,88 @@
 import RNTurboiOS
 import UIKit
 
-class RNVisitableView: UIView, SessionSubscriber {
+class RNVisitableView: UIView, RNSessionSubscriber {
   var id: UUID = UUID()
+  @objc var sessionHandle: NSString? = nil
+  @objc var applicationNameForUserAgent: NSString? = nil
   @objc var url: NSString = "" {
     didSet {
-      if (oldValue != "") {
-        self.controller?.visitableURL = URL(string: String(url))!
-        getSession()?.visit(controller!)
-      }
+      controller.visitableURL = URL(string: String(url))
+      performVisitOnUrlChange(oldUrl: oldValue)
```

Comment:
I know that we wanted to avoid checking the old value, however there is a problem with `replace` turbo action. On the RN side when `replace` action arrives, we call `setParams({url})` on react-navigation screen. Then this change is propagated to the `RNVisitableView` and no other Turbo session callback is being called. I was thinking that we could change this behavior, but we shoud maintain this communication order that we have today (proposal -> RN executes callback and propagates this to useWebviewNavigateHook -> do the visit on a native side). We could just add a line of code to the native onVisitProposal and check if action is equal to `replace` and call `turboSession.visit(...)` but we get two drawbacks:
1. We would no longer support defining custom behavior on RN side for `replace` event
2. We need to call `controller.visitableURL = URL(string: String(proposal.url))` before visit in the native onVisitProposal and we  also wanted to set it in `url didSet` only.